terra-app-setup-chart/README.md

```diff
@@ -3,12 +3,69 @@
 helm install  --set serviceAccount.annotations.gcpServiceAccount="<value>" -n <namespace> <release-name> ./terra-app-setup-chart
 ```
 # Publish Chart
-At project root, run the following commands
+For publishing a new version to prod, at project root, run the following commands.
+Note for these changes to take effect in leonardo you will need to update the version in the Dockerfile and reference.conf 
 
 ```
 helm package terra-app-setup-chart
 mkdir terra-app-setup-chart/repo
-mv terra-app-setup-0.0.1.tgz terra-app-setup-chart/repo
-helm repo index terra-app-setup-chart/repo/ --url https://storage.googleapis.com/terra-app-setup-chart
-gsutil cp -r terra-app-setup-chart/repo/* gs://terra-app-setup-chart
-```
\ No newline at end of file
+gsutil cp -r gs://terra-app-setup-chart terra-app-setup-chart/repo/
```

Comment:
so you'll need to download what's in the bucket first?

Reply:
Yeah, that's to merge the remote `index.yaml` with the new version

Reply:
is it enough to only download the remote `index.yaml` alone?

Reply:
Yeah, I think it is. Probably better than overwritting everything 👍